recipe/meta.yaml

```diff
@@ -78,19 +82,10 @@ outputs:
       commands:
         # skip test_url_is_accessible instead of hitting 20+ servers per run, since
         # each server might be occasionally unresponsive and end up failing our CI
-        - pytest -k "not test_url_is_accessible" test/
-
-  # for compatibility with (& ease of migration from) old packages in the pytorch channel, see
-  # https://anaconda.org/pytorch/torchvision-cpu/files
-  - name: torchvision-cpu
```

Comment:
are we sure this is ok to remove? maybe a migrator is necessary to replace torchvision-cpu with torchvision and pytorch-cpu

Reply:
Good point, but there [are no](https://github.com/search?q=org%3Aconda-forge+torchvision-cpu&type=code) dependent packages in conda-forge either.